### DISCUSSION TITLE
Pose Dividers On Event Logs

### DISCUSSION THREAD

**Discussion by thedreadgoose** [Arx-Game/arxcode#372] [Ideas]
I have a terrible memory. I also have an unfortunate lack of reading comprehension for someone who plays a text-based game. Sometimes when I reread event logs, I get confused as to where one player's pose ends and another begins, and vice versa, since there are no demarcations between said poses. 

It'd be great if website event logs could have a horizontal divider between player poses, in the same way that flashbacks are displayed. Maybe this could be an option that could be switched on or off, just in case players would rather view logs in the way they're currently displayed and formatted, without dividers.

**Comment by TehomCD** [comment]
Yeah, part of the issue right now is that the logs don't have any metadata to make that easy, unlike flashbacks. It's something I want to do, though.
